lgtm.yml

```diff
@@ -0,0 +1,18 @@
+queries:
+- exclude: py/insecure-default-protocol
+  # For Python>=3.6, ssl.SSLContext by default uses secure settings.
+  # See https://docs.python.org/3.6/library/ssl.html#ssl.SSLContext
+extraction:
+  javascript:
+    index:
+      exclude:
+        - ".git"
+        - ".github"
+        - "aiosmtpd"
+        - "build"
+        - "examples"
+        - "htmlcov"
+      filters:
+        - exclude: "**/*.js"
+        - exclude: "**/*.css"
+        - exclude: "**/*.html"
```

Comment:
Meant to disable LGTM checking of JavaScript. Either doesn't work, or needs to be merged first to be effective.